### DISCUSSION TITLE
Issues with Mobile navbar toggle

### DISCUSSION THREAD

**Discussion by kcits** [MasaCMS/MasaCMS#108] [Q&A]
I have installed Masa andh ave about 10+ sites running on it.  All are using the default theme from download.  All of them have the same issue with the Mobile navbar Toggle not working.  I am not getting an error in the console.  I tested just in case and added jquery to one theme to see if that was the issue (comparing Masa to a Mura install) and the didn't fix the issue.  I also tried putting Mura's toggle code in the navbar instead of Masa's and that didn'tfi x it either.  I also downloaded the Masa Boot 5 theme from github and tried that, and it didn't work either.  I see the MasaCMS uses a completely different mobile navbar type toggling system.
Is there a fix for this or does anyone have this working so I can see what the code should look like?

**Comment by kcits** [comment]
I found a fix.  If you remove the following in the html_foot.cfm:
`<script src="#$.siteConfig('themeAssetPath')#/assets/js/theme.js"></script>`

and Replace with (downloading Bootstrap 5 and putting it in the right place):
```
<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.5/dist/umd/popper.min.js" integrity="sha384-Xe+8cL9oJa6tN/veChSP7q+mnSPaj5Bcu9mPX5F5xIGE0DVittaqT5lorf0EI7Vk" crossorigin="anonymous"></script>
<script src="#$.siteConfig('themeAssetPath')#/assets/bootstrap-5-2-0-dist/js/bootstrap.min.js"></script>
```

And moved the following in the html_head.cfm to be above the Bootstrap:
`<link rel="stylesheet" href="#$.siteConfig('themeAssetPath')#/assets/css/styles.min.css">`

And Added:
`<link rel="stylesheet" href="#$.siteConfig('themeAssetPath')#/assets/bootstrap-5-2-0-dist/css/bootstrap.min.css">`

And for good measure I changed/updated this line to 7.3:
`#$.outputMuraCSS(version="7.3", includeskin=true)#`

**Comment by guustnieuwenhuis** [comment]
I've created an issue to fix this in the MasaBootstrap5 repo: https://github.com/MasaCMS/MasaBootstrap5/issues/2
